app/services/cloud_tenant_dashboard_service.rb

```diff
@@ -0,0 +1,53 @@
+class CloudTenantDashboardService < EmsCloudDashboardService
+  include Mixins::CheckedIdMixin
+
+  def initialize(tenant, controller, klass)
+    @ems_id = tenant.ext_management_system.id
+    @ems = find_record_with_rbac(klass, @ems_id) if @ems_id.present?
+    @tenant_id = tenant.id
+    @conroller = controller
+  end
+
+  def attributes_data
+    attributes = %i(cloud_object_store_containers miq_templates vms security_groups cloud_networks cloud_subnets network_ports cloud_volumes)
+
+    attr_icon = {
+      :cloud_object_store_containers    => "ff ff-cloud-object-store",
+      :miq_templates                    => "fa fa-database",
+      :vms                              => "pficon pficon-virtual-machine",
+      :security_groups                  => "pficon pficon-cloud-security",
+      :cloud_networks                   => "ff ff-cloud-network",
+      :cloud_subnets                    => "pficon pficon-network",
+      :network_ports                    => "ff ff-network-port",
+      :cloud_volumes                    => "pficon pficon-volume"
+    }
+
+    attr_url = {
+      :cloud_object_store_containers    => "cloud_object_store_containers",
+      :miq_templates                    => "images",
+      :vms                              => "instances",
+      :security_groups                  => "security_groups",
+      :cloud_networks                   => "cloud_networks",
+      :cloud_subnets                    => "cloud_subnets",
+      :network_ports                    => "network_ports",
+      :cloud_volumes                    => "cloud_volumes"
+    }
+
+    attr_hsh = {
+      :cloud_object_store_containers    => "Cloud Object Store Containers",
+      :miq_templates                    => "Images",
+      :vms                              => "Instances",
+      :security_groups                  => "Security Groups",
+      :cloud_networks                   => "Cloud Networks",
+      :cloud_subnets                    => "Cloud Subnets",
+      :network_ports                    => "Network Ports",
+      :cloud_volumes                    => "Cloud Volumes"
+    }
+
+    format_data('cloud_tenant', attributes, attr_icon, attr_url, attr_hsh)
+  end
+
+  def get_url(ems_type, ems_id, attr_url)
+    "/#{ems_type}/show/#{@tenant_id}?display=#{attr_url}"
```

Comment:
This doesn't make sense.

Either `ems_type` is not supposed to be anything related to ems, or `@tenant_id` is not a valid id. The ems_* controllers expect ems_id, not cloud_tenant_id.